lib/Compilation.js

```diff
@@ -1917,7 +1917,6 @@ class Compilation {
 			for (let indexChunk = 0; indexChunk < chunks.length; indexChunk++) {
 				const iteratedChunk = chunks[indexChunk];
 				chunkGroup.removeChunk(iteratedChunk);
-				chunkGroup.removeParent(iteratedChunk);
```

Comment:
Another bug here. `ChunkGroup#removeParent` is for `ChunkGroup`, not chunks.

Reply:
:+1: